spec/acceptance/apt_key_provider_spec.rb

```diff
@@ -42,7 +49,7 @@
 
           apply_manifest(pp, :catch_failures => true)
           apply_manifest(pp, :catch_changes => true)
-          shell("apt-key list | grep #{PUPPETLABS_GPG_KEY_ID}")
+          shell("apt-key adv --list-keys --with-colons --fingerprint | grep #{PUPPETLABS_GPG_KEY_FINGERPRINT}")
```

Comment:
I found the docs for shared_examples: http://www.relishapp.com/rspec/rspec-core/v/2-11/docs/example-groups/shared-examples

I'm going to leave such a refactoring to someone who knows the tools better.  I'll change the shell commands to use a variable though.